addons/account/models/account_move.py

```diff
@@ -1876,6 +1876,10 @@ def post(self):
         for move in self:
             if move.auto_post and move.date > fields.Date.today():
                 raise UserError(_("This move is configured to be auto-posted on {}".format(move.date.strftime(self.env['res.lang']._lang_get(self.env.user.lang).date_format))))
+
+            if move.partner_id not in move.message_partner_ids:
+                move.message_subscribe([move.partner_id.id])
```

Comment:
`move.partner_id` or `move.commercial_partner_id` ? probably both, no?

Reply:
Updated to add both